pkg/apis/minio.min.io/v2/constants.go

```diff
@@ -232,10 +232,10 @@ const LogAuditTokenKey = "LOGSEARCH_AUDIT_AUTH_TOKEN"
 
 // LogQueryTokenKey is the k8s secret/environment variable key name referring to
 // the token used to perform search query on audit logs persisted.
-const LogQueryTokenKey = "LOGSEARCH_QUERY_AUTH_TOKEN"
+const LogQueryTokenKey = "MINIO_QUERY_AUTH_TOKEN"
```

Comment:
Do we really need to rename these? now we will need to also plan an upgrade path for secrets @nitisht @harshavardhana

Reply:
We do not need to rename these, but we need to make sure to set the right environment variables for MinIO deployment when its considered an "upgrade"

Reply:
Also the ENV is wrong 

```
~ git grep MINIO_LOG
cmd/common-main.go:     if value := env.Get("MINIO_LOG_QUERY_URL", ""); value != "" {
cmd/common-main.go:             if value := env.Get("MINIO_LOG_QUERY_AUTH_TOKEN", ""); value != "" {
```

@nitisht did you test this change because it wouldn't work with logsearch API when auth token is configured.